tfx/components/example_gen/base_example_gen_executor_test.py

```diff
@@ -85,6 +85,17 @@ def setUp(self):
                                            'data_tfrecord-00000-of-00001.gz')
     self._eval_output_file = os.path.join(examples.uri, 'eval',
                                           'data_tfrecord-00000-of-00001.gz')
+    
+    # Create exec proterties for output splits.
+    self.exec_properties = {
```

Comment:
let's use _exec_properties (private/protected) instead of exec_properties (public)